api/net/Areas/Services/Controllers/ConnectionController.cs

```diff
@@ -52,12 +53,11 @@ public ConnectionController(IConnectionService serviceConnection, IOptions<JsonS
     [HttpGet("{id}")]
     [Produces(MediaTypeNames.Application.Json)]
     [ProducesResponseType(typeof(ConnectionModel), (int)HttpStatusCode.OK)]
-    [ProducesResponseType((int)HttpStatusCode.NoContent)]
+    [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
     [SwaggerOperation(Tags = new[] { "Connection" })]
     public IActionResult FindById(int id)
     {
-        var result = _serviceConnection.FindById(id);
-        if (result == null) return new NoContentResult();
+        var result = _serviceConnection.FindById(id) ?? throw new NoContentException();
```

Comment:
Switched to exception to return a BadRequest to keep it aligned with all other implementations.